client/src/components/LayerManager.vue

```diff
@@ -20,110 +17,92 @@ import { FrameDataTrack } from 'vue-media-annotator/layers/LayerTypes';
 import TextLayer from 'vue-media-annotator/layers/TextLayer';
 import Track, { TrackId } from 'vue-media-annotator/track';
 import { geojsonToBound } from 'vue-media-annotator/utils';
-import { StateStyles, TypeStyling } from 'vue-media-annotator/use/useStyling';
+
+import {
+  useIntervalTree,
+  useTrackMap,
+  useTracks,
+  useSelectedTrackId,
+  useTypeStyling,
+  useEditingMode,
+  useVisibleModes,
+  useSelectedKey,
+  useStateStyles,
+} from 'vue-media-annotator/provides';
 
 /** LayerManager is a component intended to be used as a child of an Annotator.
  *  It provides logic for switching which layers are visible, but more importantly
  *  it maps Track objects into their respective layer representations.
  *  LayerManager emits high-level events when track features get selected or updated.
  */
 export default defineComponent({
-  props: {
-    trackMap: {
-      type: Map as PropType<Map<TrackId, Track>>,
-      required: true,
-    },
-    tracks: {
-      type: Object as PropType<Ref<Track[]>>,
-      required: true,
-    },
-    intervalTree: {
-      type: Object as PropType<IntervalTree>,
-      required: true,
-    },
-    selectedTrackId: {
-      type: Object as PropType<Ref<TrackId>>,
-      required: true,
-    },
-    typeStyling: {
-      type: Object as PropType<Ref<TypeStyling>>,
-      required: true,
-    },
-    stateStyling: {
-      type: Object as PropType<StateStyles>,
-      required: true,
-    },
-    editingMode: {
-      type: Object as PropType<Ref<false|EditAnnotationTypes>>,
-      required: true,
-    },
-    visibleModes: {
-      type: Object as PropType<Ref<EditAnnotationTypes[]>>,
-      required: true,
-    },
-    selectedKey: {
-      type: Object as PropType<Ref<string>>,
-      required: true,
-    },
-  },
-
   setup(props, { emit }) {
+    const intervalTree = useIntervalTree();
+    const trackMap = useTrackMap();
+    const tracksRef = useTracks();
+    const selectedTrackIdRef = useSelectedTrackId();
+    const typeStylingRef = useTypeStyling();
+    const editingModeRef = useEditingMode();
+    const visibleModesRef = useVisibleModes();
+    const selectedKeyRef = useSelectedKey();
+    const stateStyling = useStateStyles();
+
     const annotator = inject('annotator') as Annotator;
-    const frameNumber = computed(() => annotator.frame);
+    const frameNUmberRef = computed(() => annotator.frame);
```

Comment:
```suggestion
    const frameNumberRef = computed(() => annotator.frame);
```

Along with all subsequent uses of this variable